stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetAddCardFragment.kt

```diff
@@ -21,8 +27,34 @@ internal class PaymentSheetAddCardFragment(
         )
     }
 
-    override fun onGooglePaySelected() {
-        sheetViewModel.checkout()
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val config = arguments?.getParcelable<FragmentConfig>(
+            BaseSheetActivity.EXTRA_FRAGMENT_CONFIG
+        )
+        val shouldShowGooglePayButton = config?.let {
+            config.isGooglePayReady && config.paymentMethods.isEmpty()
```

Comment:
Why do we need to check for `config.paymentMethods.isEmpty()` on this fragment?

Reply:
How it works on iPhone on the complete workflow is that if the user has payment methods it shows up on the list.  Clicking add fragment from the list doesn't show the google pay button.

The google pay button is only shown on the add sheet if there is no customer or a new customer (with no payment methods)

Reply:
Ah this is helpful, thanks.